clients/go/pkg/commands/throwError_command.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package commands
+
+import (
+	"context"
+	"github.com/zeebe-io/zeebe/clients/go/pkg/pb"
+	"time"
+)
+
+type ThrowErrorCommandStep1 interface {
+	JobKey(int64) ThrowErrorCommandStep2
+}
+
+type ThrowErrorCommandStep2 interface {
+	ErrorCode(string) ThrowErrorCommandStep3
+}
+
+type ThrowErrorCommandStep3 interface {
+	ErrorMessage(string) DispatchThrowErrorCommand
```

Comment:
The error message should be optional. I'm not sure but it seems that it is currently required, right?